src/components/PurchaseOrder/PO.js

```diff
@@ -90,20 +103,20 @@ class PO extends Component {
   }
 
   getData() {
-    return _.get(this.props, ['resources', 'order', 'records', 0], null);
+    return get(this.props, ['resources', 'order', 'records', 0], null);
   }
 
   onToggleSection({ id }) {
     this.setState((curState) => {
-      const newState = _.cloneDeep(curState);
+      const newState = cloneDeep(curState);
       newState.sections[id] = !curState.sections[id];
       return newState;
     });
   }
 
   handleExpandAll(obj) {
     this.setState((curState) => {
-      const newState = _.cloneDeep(curState);
+      const newState = cloneDeep(curState);
```

Comment:
There is no need to clone current state. Please, just return the object with changed props. In current case it would be `{ sections }`.

Reply:
@Yurii-Danylenko @VictorSoroka1 Guys. here is the root of this code, maybe it makes sense to change this readme? https://github.com/folio-org/stripes-components/blob/master/lib/Accordion/readme.md#controlled